sagemaker_neo_compilation_jobs/imageclassification_caltech/Image-classification-fulltraining-highlevel-neo.ipynb

```diff
@@ -7,9 +7,10 @@
     "# Model Optimization with an Image Classification Example\n",
```

Comment:
<p>typo:</p><p> </p><p>We will use SageMaker Neo's <code>compile_model()</code> API while <strong>specifing</strong> <code>MXNet</code> as the framework and the version to optimize the model. When calling this API, we also specify the target instance family, correct input shapes for the model and the S3 location to which the compiled model artifacts would be stored. For this example, we will choose <code>ml_c5</code> as the target instance family.</p><p>To</p><p>We will use SageMaker Neo's <code>compile_model()</code> API while <strong>specifying</strong> <code>MXNet</code> as the framework and the version to optimize the model. When calling this API, we also specify the target instance family, correct input shapes for the model and the S3 location to which the compiled model artifacts would be stored. For this example, we will choose <code>ml_c5</code> as the target instance family.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1625/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
done